build.gradle.kts

```diff
@@ -0,0 +1,52 @@
+
+plugins {
+    id("com.hivemq.extension")
+}
+
+repositories {
+    mavenCentral()
+}
+
```

Comment:
```suggestion
```
Unnecessary, as the plugin adds maven central automatically.
If you want it for demonstration purposes, please move the `repositories` block down, above the `dependencies` block